osu.Game/Graphics/Containers/ParallaxContainer.cs

```diff
@@ -34,9 +37,19 @@ public ParallaxContainer()
         protected override Container<Drawable> Content => content;
 
         [BackgroundDependencyLoader]
-        private void load(UserInputManager input)
+        private void load(UserInputManager input, OsuConfigManager config)
         {
             this.input = input;
+
+            config.GetBindable<bool>(OsuConfig.MenuParallax).ValueChanged += delegate

```

Comment:
set the local parallaxEnabled here and then you won't have to every time in the ValueChanged method.